controllers/dscinitialization/utils.go

```diff
@@ -66,9 +66,11 @@ func (r *DSCInitializationReconciler) createOdhNamespace(ctx context.Context, ds
 			r.Log.Error(err, "Unable to fetch namespace", "name", name)
 			return err
 		}
-	} else if (dscInit.Spec.Monitoring.ManagementState == operatorv1.Managed) && dscInit.Spec.Monitoring.Namespace == name {
+	} else if dscInit.Spec.Monitoring.ManagementState == operatorv1.Managed {
+		r.Log.Info("Patching application namespace for Managed cluster", "name", name)
+		lablePatch := `{"metadata":{"labels":{"openshift.io/cluster-monitoring":"true","pod-security.kubernetes.io/enforce":"baseline","opendatahub.io/generated-namespace": "true"}}}` //nolint
```

Comment:
Did you mean `labelPatch`?

Reply:
correct, this is a typo for the variable name